src/cookbooks/data_modeling/where_to_store.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+
+Mozilla’s current setup allows for the implementation of data modeling and business logic in different locations e.g BigQuery or Looker. The purpose of this document is to provide guidelines to decide where to store a new asset and to review pull requests that implement them.
+
+For the purpose of this documentation, the analytics data model is defined as the set of data assets designed to collect meaningful data from our raw datasets and structure it for an efficient understanding and analysis of our products, business processes and events.
+
+## What to check in a Pull Request in Looker
+
+When reviewing a PR first check if the data required is already implemented in an existing asset ( fully or partially) and prefer extending and existing dataset, over creating a new one or moving the calculation to Looker.
+
+- Check the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl) for an existing dataset with the required data (table, aggregate, view, materialized view)
+- Check the [looker-hub](https://github.com/mozilla/looker-hub) repository for the availability of the table or view in Looker.
+- Check the [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b) repository for existing Explores in Looker.
+
+We want to avoid merging a PR in one of the Looker repositories that:
+
+- Includes the definition of a core metric or the implementation of business logic e.g. the [type of search](https://github.com/mozilla/bigquery-etl/blob/a3e59f90326816a2ecaaa3e9d5b57fe9552f7d70/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql#L781) or the [calculation of DAU or visited URIs](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_exact_mau28_by_dimensions_v1/query.sql).
+- Implements an analysis that may need to be replicated in the future, applies to different browsers or to different scenarios.
+- Includes a transformation that needs to be tested and health checked.
+- Includes a definition that is already implemented in a BigQuery dataset. If it is partially implemented e.g. not all the dimensions are available, the recommendation is to extend the dataset instead of moving the calculation to Looker.
+
+For these cases, the recommendation in the PR is to move the implementation to bigquery-etl.
```

Comment:
If you have any examples of things that should be moved according to this list, it might be worth tracking and linking that somewhere (unless we're already fully compliant). Some closed or reverted PRs linked as examples would be useful too if they exist

Reply:
Since we haven't planned yet for migration of existing analysis based on these guidelines, the full list of examples is not collected. We will need it before starting a migration of existing analysis. For now I added some examples below in (#What to Store in Looker).